### PR TITLE
refactor: Update the README's

### PR DESCRIPTION
**Pull request by TeamWolfyta**
Update README's to have a little documentation... also update the main README to look similar to the main README of https://github.com/oceanroleplay/discord.ts

### CODE REVIEW

**Review by VictoriqueMoe — APPROVED** [review]

**Comment by TeamWolfyta** [comment]
https://discord.com/channels/874802018361950248/881450679623897118/1006140732920299531 Context by the way.

File: 1-starter/README.md [hunk 1]
```diff
@@ -36,18 +36,60 @@
...
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Join and ask in our [Discord](https://discord-ts.js.org/discord)
 
-# Thank you
+## 💖 Thank you 💖
 
-Show your support for [discordx](https://www.npmjs.com/package/discordx) by giving us a star on [github](https://github.com/oceanroleplay/discord.ts).
+If you feel like supporting us in making [Discordx](https://www.npmjs.com/package/discordx), Why not give us a star on [GitHub](https://github.com/oceanroleplay/discord.ts)?
```
Comment (oceanroleplay) [inline 1-starter/README.md:95]: I would like this to revert

File: 1-starter/README.md [hunk 2]
```diff
@@ -36,18 +36,60 @@
...
+
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
+
+## ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Join and ask in our [Discord](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline 1-starter/README.md:91]: [Discord server]

File: README.md [hunk 3]
```diff
@@ -12,18 +54,17 @@ A collection of starter bot templates used by [create-discordx](https://www.npmj
...
-# ☎️ Need help?
+## ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Join and ask in our [Discord](https://discord-ts.js.org/discord)
 
-# Thank you
+## 💖 Thank you 💖
```
Comment (oceanroleplay) [inline README.md:68]: I would like to keep, H1 heading format for all, so revert this as well.

File: 1-starter/README.md [hunk 4]
```diff
@@ -36,18 +36,60 @@
...
+## 📜 Documentation
+
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
+
+## ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
```
Comment (oceanroleplay) [inline 1-starter/README.md:49]: revert this as well

File: 1-starter/README.md (hunk 1 above)
Comment (TeamWolfyta) [reply 1-starter/README.md:95]: Ye but new one sounds less stiff

File: 1-starter/README.md (hunk 2 above)
Comment (TeamWolfyta) [reply 1-starter/README.md:91]: Used Discord instead of Discord Server because people refer to their Discord server as Discord... But either works. Will revert it.

File: README.md (hunk 3 above)
Comment (TeamWolfyta) [reply README.md:68]: If I remember correctly, H1 is used as page heading, You wanna use H2 for non page headings.

File: 1-starter/README.md (hunk 4 above)
Comment (TeamWolfyta) [reply 1-starter/README.md:49]: Sounds too stiff...

File: 1-starter/README.md (hunk 1 above)
Comment (VictoriqueMoe) [reply 1-starter/README.md:95]: I agree with @TeamWolfyta, this new one sounds less forceful

File: README.md (hunk 3 above)
Comment (VictoriqueMoe) [reply README.md:68]: H2 is a sub heading. H2 is used for headings inside of H1

File: 1-starter/README.md (hunk 4 above)
Comment (VictoriqueMoe) [reply 1-starter/README.md:49]: > Sounds too stiff...

Keep it in, but just reword it to sound more inviting. I agree with Harry we need to have our discord server in

Comment (TeamWolfyta) [reply 1-starter/README.md:49]: This never got removed, It got changed it too `Join and ask in our Discord Server` which sounds like the best way to put it without changing it to much.

File: 1-starter/README.md (hunk 1 above)
Comment (oceanroleplay) [reply 1-starter/README.md:95]: I agree, we may need more polite line, but I am not agree on "feel"

Comment (TeamWolfyta) [reply 1-starter/README.md:95]: I can think of something better then, what about `Why not support us in making DiscordX by giving us a star on Github`

File: README.md (hunk 3 above)
Comment (TeamWolfyta) [reply README.md:68]: Got ya, I will fix it to use H1 again then

File: 1-starter/README.md (hunk 1 above)
Comment (oceanroleplay) [reply 1-starter/README.md:95]: revert for now, make another PR for this, but I still not happy with new suggestion.

File: 1-starter/README.md (hunk 4 above)
Comment (oceanroleplay) [reply 1-starter/README.md:49]: Revert

File: 1-starter/README.md [hunk 5]
```diff
@@ -36,18 +36,60 @@
...
+
+# 💻 Production
+
+```
+npm install --production
+npm run build
+npm run start
+```
+
+## 🐋 Docker (Recommended)
```
Comment (oceanroleplay) [inline 1-starter/README.md:60]:
```suggestion
# 🐋 Docker
```

File: 1-starter/README.md [hunk 6]
```diff
@@ -36,18 +36,60 @@
...
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
 
 # ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Ask in [Discord Server](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline 1-starter/README.md:91]:
```suggestion
- Ask in **[Discord Server](https://discord-ts.js.org/discord)**
```

File: 2-starter-with-api/README.md [hunk 7]
```diff
@@ -36,18 +36,60 @@
...
+
+# 💻 Production
+
+```
+npm install --production
+npm run build
+npm run start
+```
+
+## 🐋 Docker (Recommended)
```
Comment (oceanroleplay) [inline 2-starter-with-api/README.md:60]:
```suggestion
# 🐋 Docker
```

File: 2-starter-with-api/README.md [hunk 8]
```diff
@@ -36,18 +36,60 @@
...
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
 
 # ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Ask in [Discord Server](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline 2-starter-with-api/README.md:91]:
```suggestion
- Ask in **[Discord Server](https://discord-ts.js.org/discord)**
```

File: 3-blank/README.md [hunk 9]
```diff
@@ -36,18 +36,60 @@
...
+
+# 💻 Production
+
+```
+npm install --production
+npm run build
+npm run start
+```
+
+## 🐋 Docker (Recommended)
```
Comment (oceanroleplay) [inline 3-blank/README.md:60]:
```suggestion
# 🐋 Docker
```

File: 3-blank/README.md [hunk 10]
```diff
@@ -36,18 +36,60 @@
...
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
 
 # ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Ask in [Discord Server](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline 3-blank/README.md:91]:
```suggestion
- Ask in **[Discord Server](https://discord-ts.js.org/discord)**
```

File: 4-music-player-ytdl/README.md [hunk 11]
```diff
@@ -1,29 +1,95 @@
...
-- `npm run build`
-- `set BOT_TOKEN=<your bot token>`
+```
+npm install --production
+npm run build
+npm run start
+```
 
-  if you don't have token yet than create one at [discord developer portal](https://discord.com/developers/)
+## 🐋 Docker (Recommended)
```
Comment (oceanroleplay) [inline 4-music-player-ytdl/README.md:60]:
```suggestion
# 🐋 Docker
```

File: 5-music-player-lavalink/README.md [hunk 12]
```diff
@@ -36,18 +36,60 @@
...
+
+# 💻 Production
+
+```
+npm install --production
+npm run build
+npm run start
+```
+
+## 🐋 Docker (Recommended)
```
Comment (oceanroleplay) [inline 5-music-player-lavalink/README.md:60]:
```suggestion
# 🐋 Docker
```

File: 5-music-player-lavalink/README.md [hunk 13]
```diff
@@ -36,18 +36,60 @@
...
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
 
 # ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Ask in [Discord Server](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline 5-music-player-lavalink/README.md:91]:
```suggestion
- Ask in **[Discord Server](https://discord-ts.js.org/discord)**
```

File: README.md [hunk 14]
```diff
@@ -14,16 +56,15 @@ A collection of starter bot templates used by [create-discordx](https://www.npmj
...
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
 
 # ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
+- Ask in [Discord Server](https://discord-ts.js.org/discord)
```
Comment (oceanroleplay) [inline README.md:66]:
```suggestion
- Ask in **[Discord Server](https://discord-ts.js.org/discord)**
```

**Review by oceanroleplay — APPROVED** [review]
